ovos_utils/file_utils.py

```diff
@@ -86,7 +86,18 @@ def resolve_ovos_resource_file(res_name: str) -> Optional[str]:
         filename = join(core_root, "res", res_name)
         if os.path.isfile(filename):
             return filename
-    except:
+    except ImportError:
+        pass
+
+    # ovos-shell doesn't know about core resource locations
+    # check the shell plugin
```

Comment:
as mentioned in chat this method is only supposed to look for SYSTEM qml files in ovos-gui package  (and mycroft/res module for compat)

non SYSTEM should not be scanned here

Reply:
Removed the plugin and updated to handle a passed extra directory